compiler/luci-interpreter/src/KernelsToBuild.lst

```diff
@@ -0,0 +1,67 @@
+REGISTER_KERNEL(Add)
```

Comment:
Is it desired format?

If I remember correctly, there were three of them:
- REGISTER_KERNEL(Add)
- REGISTER_KERNEL(ADD)
- REGISTER_KERNEL(ADD, Add)

Remember we are going to use this file for kernel builder compile time configuration.

Reply:
Nevermind, this is fine